package.json

```diff
@@ -46,10 +39,22 @@
   },
   "devDependencies": {
```

Comment:
Do we have anything to make sure all dev dependencies get installed?  Or does that happen automatically with `npm install`?